pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -712,19 +705,31 @@ func (s *Summarizer) printReport() error {
 		}
 	}
 
-	bytes, err := json.MarshalIndent(s.fullReport, "", " ")
+	b, err := json.MarshalIndent(s.fullReport, "", " ")
 	if err != nil {
-		return fmt.Errorf("error marshalling report: %v", err)
+		return fmt.Errorf("error marshalling report: %s", err.Error())
 	}
 
-	txt := string(bytes)
-	logrus.Debugf("json txt: %+v", txt)
+	logrus.Debugf("json txt: %s", b)
 	return nil
 }
 
 func printCheck(check *kb.Check) {
+	logrus.Debugf("check: ")
```

Comment:
We need not print all these individually and just print the object as done by this previous commit: https://github.com/rancher/security-scan/pull/25/files